### ISSUE TITLE
Cannot plot winner genome plots

### ISSUE THREAD

**Issue by yqy6611** [MrOlm/drep#214]
Hi all, 

I encountered an issue on plotting:
***************************************************
    ..:: dRep dereplicate Step 5. Analyze ::..
***************************************************
    
making plots 1, 2, 3, 4, 5, 6
Plotting primary dendrogram
Plotting secondary dendrograms
Plotting MDS plot
Plotting scatterplots
Plotting bin scorring plot
Failed to make plot #5: list index out of range
Plotting winning genomes plot...
Failed to make plot #6: list index out of range

Any solutions? Thanks!

**Comment by GGasch** [comment]
Hi all,

I ran into the same issues while running a dereplication job. 

I don't know if it help, but here are the error messages :

Failed to make plot #5: list index out of range
Traceback (most recent call last):
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 108, in d_analyze_wrapper
    plot_binscoring_from_wd(wd, plot_dir, **kwargs)
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 369, in plot_binscoring_from_wd
    plot_winner_scoring_complex(Wdb, Sdb, Cdb, Gdb, plot_dir = plot_dir, **kwargs)
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 734, in plot_winner_scoring_complex
    g.annotate("{0:.1f}".format(vals[i]), (p.get_width(), p.get_y()+(p.get_height()/1.1) ), fontsize=8)
                                ~~~~^^^
IndexError: list index out of range
Plotting winning genomes plot...
Failed to make plot #6: list index out of range
Traceback (most recent call last):
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 117, in d_analyze_wrapper
    plot_winners_from_wd(wd, plot_dir, **kwargs)
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 403, in plot_winners_from_wd
    plot_winners(Wdb, Gdb, Wndb, Wmdb, Widb, plot_dir = plot_dir, **kwargs)
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 840, in plot_winners
    _make_scoring_plot(d,bars,**kwargs)
  File "/home/guest-biom/miniconda3/envs/dRep/lib/python3.12/site-packages/drep/d_analyze.py", line 1384, in _make_scoring_plot
    g.annotate("{0:.1f}".format(vals[i]), (p.get_width(), p.get_y()+(p.get_height()/1.1) ), fontsize=8)

**Comment by Phi1ippe** [comment]
Hello, 
I'm having the same problem. Have you solved your last problem? If so, how please.
Thanks in advance

**Comment by MrOlm** [comment]
Fixed in version 3.5.0
